misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/DiagramUtil.java

```diff
@@ -53,6 +55,25 @@ public static JsonElement getSyntaxTreeJSON(Document srcFile, SemanticModel sema
         return syntaxTreeJson;
     }
 
+    /**
+     * Get the Modified JSON ST with type info for the node.
+     *
+     * @param node  {@link NonTerminalNode} The node that needs to be mapped
+     * @param semanticModel {@link SemanticModel} Semantic model for the syntax tree.
+     * @return {@link JsonObject}   ST as a Json Object
+     */
+    public static JsonElement getSyntaxTreeJSONByRange(NonTerminalNode node, SemanticModel semanticModel) {
```

Comment:
Shall we rename the method to getSyntaxTreeJSON so it will be an overloaded method? It will be cleaner.